File: filedata/entity/c#/Albumtype.cs

using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
namespace Coldairarrow.Entity.Video
{
    /// <summary>
    /// 
    /// </summary>
    [Table("album_type")]
    public class Albumtype
    {
        
        /// <summary>
        /// ID主键 
可关联Videos表中VideoTypeId字段
        /// </summary>
        [Column("Id")]
        public string Id { get; set; }
    
    
        /// <summary>
        /// 专属站点 不为null则是专属
        /// </summary>
        [Column("CompanyStyle")]
        public string CompanyStyle { get; set; }
    
    
        /// <summary>
        /// 专辑名称
        /// </summary>
        [Column("AlbumName")]
        public string AlbumName { get; set; }
    
    
        /// <summary>
        /// 专辑封面图
        /// </summary>
        [Column("AlbumImg")]
        public string AlbumImg { get; set; }
    
    
        /// <summary>
        /// 排序
        /// </summary>
        [Column("Orders")]
        public int Orders { get; set; }
    
    
        /// <summary>
        /// 状态 0=停用 1=启用
        /// </summary>
        [Column("Status")]
        public bool Status { get; set; }
    
    
        /// <summary>
        /// 专辑详情
        /// </summary>
        [Column("AlbumTitle")]
        public string AlbumTitle { get; set; }
    
    
        /// <summary>
        /// 创建时间
        /// </summary>
        [Column("CreatTime")]
        public DateTime CreatTime { get; set; }
    
    
    }
}

File: filedata/entity/java/albumType.java

package entity;
// 
public class Albumtype  extends BaseDomain{
    public Albumtype() {

    }
    
    //
    private String Id;
        
    //
    private String CompanyStyle;
        
    //
    private String AlbumName;
        
    //
    private String AlbumImg;
        
    //
    private int(10) Orders;
        
    //
    private bit(10) Status;
        
    //
    private String AlbumTitle;
        
    //
    private Date CreatTime;
        
    public String getId() {
        return Id;
    }
    public void setId(String Id) {
        this.Id = Id;
    }
            
    public String getCompanystyle() {
        return Companystyle;
    }
    public void setCompanystyle(String Companystyle) {
        this.Companystyle = Companystyle;
    }
            
    public String getAlbumname() {
        return Albumname;
    }
    public void setAlbumname(String Albumname) {
        this.Albumname = Albumname;
    }
            
    public String getAlbumimg() {
        return Albumimg;
    }
    public void setAlbumimg(String Albumimg) {
        this.Albumimg = Albumimg;
    }
            
    public int(10) getOrders() {
        return Orders;
    }
    public void setOrders(int(10) Orders) {
        this.Orders = Orders;
    }
            
    public bit(10) getStatus() {
        return Status;
    }
    public void setStatus(bit(10) Status) {
        this.Status = Status;
    }
            
    public String getAlbumtitle() {
        return Albumtitle;
    }
    public void setAlbumtitle(String Albumtitle) {
        this.Albumtitle = Albumtitle;
    }
            
    public Date getCreattime() {
        return Creattime;
    }
    public void setCreattime(Date Creattime) {
        this.Creattime = Creattime;
    }
             
    @Override
    public String toString() {
        return "album_type[Id=" + Id + ",CompanyStyle=" + CompanyStyle + ",AlbumName=" + AlbumName + ",AlbumImg=" + AlbumImg + ",Orders=" + Orders + ",Status=" + Status + ",AlbumTitle=" + AlbumTitle + ",CreatTime=" + CreatTime + "]";
    }
        
    
}
        